projects/jbig2dec/jbig2_fuzzer.dict

```diff
@@ -0,0 +1,98 @@
+# AFL dictionary for jbig2 images
+# by Sebastian Rasmussen <sebras@gmail.com>
+
+id_string="\x97\x4a\x42\x32\x0d\x0a\x1a\x0a"
```

Comment:
tip: you don't actually name tokens in dicts. 
If you just put each token on a line by itself it works. I'm not even sure what the the naming is for (only benefit I see is that it sort of documents each token in cases where each token is a byte).

Reply:
I read the AFL dictionary specification and it says that naming tokens is for documentation, they are ignored when the tokens are actually used.

I decided to label the only token I could find in the JBIG2 format that is static and at least 8 bits long. Other static tokens are shorter than 8 bits, e.g. the page information segment type which is 6 bits long (the top two bits in that byte may change depending on other things), so they are not useful as dictionary tokens.